packages/block-library/src/navigation/index.php

```diff
@@ -449,9 +449,16 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 		}
 
 		$inner_blocks = new WP_Block_List( $fallback_blocks, $attributes );
-
 	}
 
+	/**
+	 * Filter navigation block $inner_blocks.
+	 * Allows modification of a navigation block menu items.
```

Comment:
I also see now that `@since` should be added:
```suggestion
	 * Allows modification of a navigation block menu items.
	 *
	 * @sicne 6.1.0	 
```